tests/functional/adapter/conftest.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+
+@pytest.fixture
+def model_ddl(request) -> str:
+    """
+    Returns the contents of the DDL file for the model provided. Use with pytest parameterization.
+
+    Example:
+    ===
+    @pytest.mark.parametrize(
+        "model_ddl,backup_expected",
+        [("backup_is_false", False)],
+        indirect=["model_ddl"]
+    )
+    def test_setting_reflects_config_option(self, model_ddl: str, backup_expected: bool):
+        backup_will_occur = "backup no" not in model_ddl.lower()
+        assert backup_will_occur == backup_expected
+    ===
+
+    This example will return the contents of the backup_is_false DDL file, which can be referenced via model_ddl.
```

Comment:
Mind editing this comment to reference that `yield` returns the contents and does so as a string. Took me and Matt a bit to figure out what this was doing since you joined on `' '` instead of `\n'. It's functional, just a bit unexpected/unclear.

Reply:
Oh, fair. So would it make more sense to update the comment? Or to join on a newline? I can explicitly put string in the comment, but it's also in the type annotation. Let me know what makes the most intuitive sense and I'll make the change.